src/test/java/org/jclouds/vcloud/director/v1_5/domain/dmtf/EnvelopeTest.java

```diff
@@ -39,7 +39,7 @@ protected void setup() throws JAXBException {
    public void testUnmarshallEnvelope() throws JAXBException {
       Unmarshaller unmarshaller = jc.createUnmarshaller();
       unmarshaller.setEventHandler(new javax.xml.bind.helpers.DefaultValidationEventHandler());
-      File xml = new File("vcloud-director/src/test/resources/dmtf/envelope.xml");
+      File xml = new File("src/test/resources/dmtf/envelope.xml");
```

Comment:
Is this the normal way jclouds gets the xml from a resource in `src/test/resources`?

If so, then fine. But my guess was that the norm would be to load the resource `dmtf/envelope.xml`, and then to get its contents.